<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>265</width>
         <height>305</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>File Transfer</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <property name="spacing">
             <number>1</number>
            </property>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>110</height>
               </size>
              </property>
              <property name="title">
               <string>Server</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_15">
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item row="0" column="0">
                <widget class="QScrollArea" name="scrollArea_3">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>86</height>
                  </size>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_3">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>249</width>
                    <height>84</height>
                   </rect>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>85</height>
                   </size>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_13">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>1</number>
                   </property>
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_5">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_3">
                       <item row="0" column="0">
                        <widget class="QPushButton" name="btnSelectSendFile">
                         <property name="text">
                          <string>File Select</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <spacer name="horizontalSpacer_4">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item row="0" column="2">
                        <widget class="QPushButton" name="btnServerEcho">
                         <property name="text">
                          <string>Echo</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <layout class="QGridLayout" name="gridLayout_4">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>Host Name:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="tbServerHostName"/>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="label">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>Host IP:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="QLineEdit" name="tbServerIP"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Client</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <property name="leftMargin">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>1</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <property name="spacing">
                <number>1</number>
               </property>
               <item row="0" column="0">
                <widget class="QScrollArea" name="scrollArea_2">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_2">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>249</width>
                    <height>145</height>
                   </rect>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_11">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>1</number>
                   </property>
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_10">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_8">
                       <item row="0" column="0">
                        <widget class="QPushButton" name="btnSetRecvFileDir">
                         <property name="text">
                          <string>Set Directory</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <spacer name="horizontalSpacer_3">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item row="2" column="0">
                      <layout class="QGridLayout" name="gridLayout_7">
                       <item row="0" column="1">
                        <widget class="QPushButton" name="btnConnectClient">
                         <property name="text">
                          <string>Receive</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <spacer name="horizontalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <layout class="QGridLayout" name="gridLayout_6">
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>File Name:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0">
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>Host Name:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="QLineEdit" name="tbClientFileName">
                         <property name="text">
                          <string>recv_file.txt</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="1">
                        <widget class="QLineEdit" name="tbClientHostName">
                         <property name="text">
                          <string>192.168.0.16</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>Directory:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="tbClientFileDir">
                         <property name="readOnly">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>269</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="btnExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="btnAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
  </widget>
  <action name="btnExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="btnAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>icon/icons8-empty-trash-20.png</normaloff>icon/icons8-empty-trash-20.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
